docs/Growing.md

```diff
@@ -117,6 +117,139 @@ functionality and then moves on to producing code that passes these tests. There
 having good test coverage and code being designed for testability, which will in turn often prevent future changes from
 breaking old code. Much of mojo.js was developed using TDD.
 
+## Prototype
+
+Mojolicious allows you to both create a lite app and a full app. The lite app is perfect for rapid prototyping.
```

Comment:
We should use mojo.js consistently rather than Mojolicious
```suggestion
mojo.js allows you to both create a lite app and a full app. The lite app is perfect for rapid prototyping.
```